.eslintrc

```diff
@@ -0,0 +1,22 @@
+// Reference: eslint.org/docs/user-guide/configuring/configuration-files
+
+{
+  "root": true,
+  "extends": [
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "parserOptions": { "project": ["./tsconfig.json"] },
```

Comment:
Make a copy of tsconfig file with the `ts-lint.json` name. and put this inside the "parserOptions"